### DISCUSSION TITLE
现在要5级才能答题,未5级就网页答题最后会提示无效问卷

### DISCUSSION THREAD

**Discussion by razilong** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#20] [Show and tell]

**Comment by MlgmXyysd** [comment]
对的，这两句就是在说无效问卷的限制
![image](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/17919858/ae36cc49-0b68-479b-a05d-51c245cc7eb5)
